package.json

```diff
@@ -29,7 +29,6 @@
     "typecheck": "tsc && tsc -p cypress",
     "validate": "run-p \"test -- --run\" lint typecheck test:e2e:run"
   },
-  "prettier": {},
   "eslintIgnore": [
```

Comment:
Since you're preferring ignore files vs. doing things in `package.json`, I'd move this to a separate file as well.

Reply:
It's my first time every seeing that line, I was just wondering what it's purpose is? I assumed it's where we put all our formatting options from? If that was the case I already created a `.prettierrc` for it

Reply:
I'm talking about `eslintIgnore`.  See https://eslint.org/docs/latest/use/configure/ignore